python_scripts/trees.py

```diff
@@ -156,10 +156,10 @@ def plot_decision_function(X, y, clf, ax=None):
 )
 
 # %% [markdown]
-# Unlike linear model, decision tree will partition the space considering a
-# single feature at a time. Let's illustrate this behaviour by having
-# a decision tree which makes a single split to partition the feature space.
-# the decision tree to make a single split to partition our feature space.
+# Unlike linear models, decision trees will partition the space by considering
+# a single feature at a time. Let's illustrate this behaviour by having
```

Comment:
Very nitpicky but I think we should be consistent in either using British (e.g., 'behaviour', 'modelling', 'organisation') or American ('behavior', 'modeling', 'organization') spelling.
Of course I support the correct and proper way to spell things - British.

Reply:
Opened issue